sc/social/like/tests/test_viewlets.py

```diff
@@ -9,10 +9,14 @@
 from sc.social.like.tests.api_hacks import set_image_field
 
 import contextlib
+import os
 import re
 import unittest
 
 
+# set the "SKIP_CODE_PROFILING" environent variable to skip profiling
```

Comment:
Where does `SKIP_CODE_PROFILING` comes from? Is it from `.travis.yml`? If so, a small doc here would be a nice addition.

Reply:
I just added that for running test locally as the profiling pollutes the test results.